### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by coquelin77**
created pull_request_template.md

### CODE REVIEW

**Comment by ClaudiaComito** [comment]
This is really helpful, thanks a lot.

File: docs/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## Description
+
+Please include a summary of the change and which issue is fixed.
```
[the remark this answers is not in the document]
Comment (coquelin77) [reply docs/pull_request_template.md]: i dont see why not. suppose we have a branch to fix a bug and in the course of fixing it, the debugger finds a solution to another bug listed in a separate issue

**Comment by coquelin77** [comment]
> I like it. Short, but includes all the necessary points.
> 
> Question regarding the "Fixes:" point; is it supposed to be a comma separated enumeration?

well i suppose that it can be any type of list. i personally dont care how they are listed so long as the issues are listed

**Comment by coquelin77** [comment]
last looks: i will merge this when i return from lunch today.

**Pull request merged**